stable/traefik/README.md

```diff
@@ -91,6 +91,7 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `serviceType`                   | A valid Kubernetes service type                                      | `LoadBalancer`                            |
 | `loadBalancerIP`                | An available static IP you have reserved on your cloud platform      | None                                      |
 | `loadBalancerSourceRanges`      | list of IP CIDRs allowed access to load balancer (if supported)      | None                                      |
+| `externalTrafficPolicy`         | Set the externalTrafficPolicy in the Service to either Cluster or Local | `Cluster`                              |
```

Comment:
The use cases for this might not be obvious to most users. It's a complicated enough answer that I don't think adding a lot of detail in-line here is advisable, but perhaps you can just link off to this bit of doc?

https://kubernetes.io/docs/tutorials/services/source-ip/

Reply:
Should I do that in the table or somewhere below the table? I think the link might be a bit too much for the table, but not sure.